smithy-smoke-test-traits/src/main/java/software/amazon/smithy/smoketests/traits/SmokeTestsTrait.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think we remove the date now, and maybe use the short form of license headers

Reply:
Removed the date, I'm not sure what the short form of license headers is though. Do you have an example?